src/styles/global.css

```diff
@@ -40,9 +40,13 @@ html {
 body {
   margin: 0;
   padding: 0;
+  overflow-x: hidden;
```

Comment:
Why is this needed? If there is accidental overflowing somewhere, we should fix it there instead.

Reply:
You right! I think i forgot to remove this while testing some stuff. I'll fix it now.